### DISCUSSION TITLE
How to translate Categories, Series, Tags without creating corresponding _index.md?

### DISCUSSION THREAD

**Discussion by TetraTheta** [hbstack/.github#30] [Q&A] [answered]
At the top of 'front page' of site, we have these icons which leads to  `mysite.com/categories`, `mysite.com/series`, `mysite.com/tags`.

![IMG1696654546](https://github.com/hbstack/.github/assets/37061801/33dc2d9a-3e9b-49f7-9dca-55539c0a9625)

And you clikc one of them, it will show you something like this:

![IMG1696654692](https://github.com/hbstack/.github/assets/37061801/6912027a-fcd5-420e-a04c-898156b95cdf)

'Categories' is not translated.

I think the only way to translate it is creating `content/categories/_index.md` and put `title: 카테고리` as front matter. Same thing applies to 'Series' and 'Tags' too.

---

And here starts my problem.

I don't have plan to use `docs` module for now. All of my content are currently under `content/blog/` directory.
```
content
├ blog
│ ├ chit-chat
│ └ game
├ categories
├ series
└ tags
```
But I want to organize my contents like this:
```
content
├ chit-chat
├ game
└ docs (if I use `docs` module later)
```
This will simplify content's URL by removing `/blog/` part from the URL. `/docs/` would be 'special case' if I use it on my site.

Since `categories`, `series`, `tags` directory don't have any actual content, I simply want to remove it. But doing that will revert my translation back to 'Categories', 'Series', 'Tags'. 

&nbsp;

So, how can I translate those 'Categories', 'Series', 'Tags' without creating placeholder `_index.md` file and its parent directory?

**Comment by razonyang** [comment, marked as answer]
No, it's impossible without creating _index.md, and specify the `title` parameter, unless you implement it by overriding the [layouts/partials/hb/modules/blog/home/taxonomies.html](https://github.com/hbstack/blog/blob/main/layouts/partials/hb/modules/blog/home/taxonomies.html) and the i18n function.
**But it won't be accepted to be merged into the module**, it makes no sense to me to do that, since the pages always be generated, even there are `_index.md` in taxonomies folder or not. Beside the taxonomies item will be missed from breadcrumb without `_index.md`.

**Reply by TetraTheta** [reply]
Yeah, I missed the breadcrumb will also be untranslated if I remove `_index.md`. 

Judging from class of 'untranslated title', I think I should modify at least [`layouts/_default/terms.html`](https://github.com/hbstack/blog/blob/main/layouts/_default/terms.html) and [`layouts/partials/hb/modules/breadcrumb/index.html`](https://github.com/hbstack/breadcrumb/blob/main/layouts/partials/hb/modules/breadcrumb/index.html) but this will result too many work for me.

I will try to use Hugo's [URL Management](https://gohugo.io/content-management/urls/) feature instead.  
Redirecting `/blog/game/` to `/game` and `/blog/chit-chat/` to `/chit-chat` will be sufficient, I guess.

Thank you for the answer!
